MC/config/PWGLF/pythia8/generator_pythia8_box.C

```diff
@@ -0,0 +1,99 @@
+#include "Pythia8/Pythia.h"
+#include "FairGenerator.h"
+#include "FairPrimaryGenerator.h"
+#include "Generators/GeneratorPythia8.h"
+#include "TRandom3.h"
+#include "TParticlePDG.h"
+#include "TDatabasePDG.h"
+#include "TMath.h"
+#include <cmath>
```

Comment:
This should be surrounded to guard the includes wrt `CLING` and `ROOTCLING` (see https://aliceo2group.github.io/simulation/docs/generators/generatorscustom.html#rootclingcling-related-problems)

Reply:
Sorry, `using namespace Pythia8;` should go in there as well

Reply:
thanks, I have just pushed the fixes. Still with the guards the test fails:

`/home/fmazzasc/alice/sw/ubuntu2004_x86-64/O2DPG/fix-local1/MC/config/PWGLF/pythia8/generator_pythia8_box.C:66:7: error: unknown type name 'Particle'
      Particle myparticle;`

Reply:
Thanks, I will take a look!

Reply:
sorry for the confusion. Of course
```cpp
#include Pythia8/Pythia.h
using namespace Pythia8;
```
must be both outside the include guards.
There are no dictionaries for those and to get the type right etc, the header should be there.

I also tested quickly now and it works like a charm.
So once, you can add this tiny fix, we are good to merge.

Reply:
ok everything clear now:) i pushed the fix, thanks!